Octokit/Models/Response/GitHubAppFromManifest.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents a GitHub application from a manifest.
+    /// https://docs.github.com/rest/apps/apps#create-a-github-app-from-a-manifest
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
```

Comment:
What's this for?

Reply:
When creating a new GitHub App using the [Manifest flow](https://docs.github.com/en/apps/sharing-github-apps/registering-a-github-app-from-a-manifest#about-github-app-manifests), GitHub redirects the user to the external API with a [`code` query parameter that the service can exchange](https://docs.github.com/en/apps/sharing-github-apps/registering-a-github-app-from-a-manifest#3-you-exchange-the-temporary-code-to-retrieve-the-app-configuration) for all the details about the new GitHub App, including the Client Id, Client Secret, auto-generated Webhook Secret, PEM, etc.

As such, it's critical to deserialize the additional values on this type and save them for the new GitHub App's backing service to use.

From GitHub's [official REST API spec](https://raw.githubusercontent.com/github/rest-api-description/main/descriptions/api.github.com/api.github.com.yaml):

```openapi
"/app-manifests/{code}/conversions":
    post:
      summary: Create a GitHub App from a manifest
      description: Use this endpoint to complete the handshake necessary when implementing
        the [GitHub App Manifest flow](https://docs.github.com/apps/building-github-apps/creating-github-apps-from-a-manifest/).
        When you create a GitHub App with the manifest flow, you receive a temporary
        `code` used to retrieve the GitHub App's `id`, `pem` (private key), and `webhook_secret`.
      tags:
      - apps
      operationId: apps/create-from-manifest
      externalDocs:
        description: API method documentation
        url: https://docs.github.com/rest/apps/apps#create-a-github-app-from-a-manifest
      parameters:
      - name: code
        in: path
        required: true
        schema:
          type: string
      responses:
        '201':
          description: Response
          content:
            application/json:
              schema:
                allOf:
                - "$ref": "#/components/schemas/integration"
                - type: object
                  properties:
                    client_id:
                      type: string
                    client_secret:
                      type: string
                    webhook_secret:
                      type: string
                      nullable: true
                    pem:
                      type: string
                  required:
                  - client_id
                  - client_secret
                  - webhook_secret
                  - pem
                  additionalProperties: true
              examples:
                default:
                  "$ref": "#/components/examples/integration-from-manifest"
        '404':
          "$ref": "#/components/responses/not_found"
        '422':
          "$ref": "#/components/responses/validation_failed_simple"
```